lib/web/apiserver.go

```diff
@@ -793,7 +793,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	// h.GET("/webapi/users/password/token/:token", h.WithLimiter(h.getResetPasswordTokenHandle))
 	h.GET("/webapi/users/*wildcard", h.handleGetUserOrResetToken)
 
-	h.PUT("/webapi/users/password/token", httplib.WithCSRFProtection(h.changeUserAuthentication))
+	h.PUT("/webapi/users/password/token", h.WithLimiter(h.changeUserAuthentication))
```

Comment:
Technically this is a behavior change, as we weren't rate limiting this endpoint at all.

I figured resetting user authentication happens a lot less frequently than users logging in, so if we're okay to rate limit `createWebSession` then it should be fine to limit `changeUserAuthentication` too.